### PR TITLE
added merge request screenshots

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: sources/start/gitlab/index.md [hunk 1]
```diff
@@ -219,17 +219,31 @@ spec:
...
+
+If you select the Expand button in the code quality section you will see the updates to the Advise from Reliably as a result making the merge request:
+
+![Code Quality Output](./images/code-quality-output.png)
+
+### Viewing  Code Quality output on the GitLab CI/CD Pipeline
+
+You can also view the Code Quality report on the GitLab CI/CD Pipeline page. On the GitLab page for your project on the left hand menu you will find the CI/CD menu, with an entry of Pipelines. Select the Pipelines button:
 
 ![Code Qaulity](./images/manifest-running.png)
```
Comment (mperrien) [inline sources/start/gitlab/index.md:237]: Typo!
```suggestion
![Code Quality](./images/manifest-running.png)
```

File: sources/start/gitlab/index.md [hunk 2]
```diff
@@ -219,17 +219,31 @@ spec:
...
 
-This manifest.yaml file has been specifically written to trigger some of Reliably's Advice and Suggestions. Adding the manifest will make a CI/CD job run:
+This manifest.yaml file has been specifically written to trigger some of Reliably's Advice and Suggestions. If you add the manifest by making a merge request, the merge request will show any changes in the code quality ouput when the Pipeline has completed:
+
+![Merge Request](./images/merge-review.png)
+
+!!!note
+    You often need to do a page refresh on the merge request page to see the code quality section
+
+If you select the Expand button in the code quality section you will see the updates to the Advise from Reliably as a result making the merge request:
```
Comment (mperrien) [inline sources/start/gitlab/index.md:229]: I find this sentence hard to understand.

File: sources/start/gitlab/index.md [hunk 3]
```diff
@@ -219,17 +219,31 @@ spec:
...
+### Viewing  Code Quality output on the GitLab CI/CD Pipeline
+
+You can also view the Code Quality report on the GitLab CI/CD Pipeline page. On the GitLab page for your project on the left hand menu you will find the CI/CD menu, with an entry of Pipelines. Select the Pipelines button:
 
 ![Code Qaulity](./images/manifest-running.png)
 
-You can view the output of the running job both when its running and when it's complete. On the Gitlab page for your project on the left hand menu you will find the CI/CD menu, with an entry of Pipelines. Select the Pipelines button, you will then see a page with your Pipeline entries, select the  status button on one of these entries:
+You can see your running and completed Pipeline jobs. You can view the job in detail  by selecting the button in the Status column:
 
 ![Code Qaulity Job](./images/code-quality-job.png)
```
Comment (mperrien) [inline sources/start/gitlab/index.md:241]: Typo!
```suggestion
![Code Quality Job](./images/code-quality-job.png)
```

File: sources/start/gitlab/index.md [hunk 4]
```diff
@@ -219,17 +219,31 @@ spec:
...
 
-If you then select the Code Quality button in the Pipeline view you will see the output of the excuting job including the advice output from the `reliably discover`:
+For a completed job, a Code Quality tab will be displayed. If you select the tab
```
Comment (mperrien) [inline sources/start/gitlab/index.md:243]:
```suggestion
For a completed job, a Code Quality tab will be displayed. If you select the tab,
```

File: sources/start/gitlab/index.md [hunk 5]
```diff
@@ -238,4 +252,4 @@ Reliably has surfaced two Suggestions:
 
 All of these Reliably Suggestions could lead to reliability problems if you were to apply this manifest to your system.
 
-Congratulations, you now have Reliably integrated into a repository on GitLab so that Reliably can start surfacing Reliability Suggestions as you evolve your code.
+Congratulations, you now integrated Reliably into a repository on GitLab so that Reliably can start surfacing Reliability Suggestions as you evolve your code.
```
Comment (mperrien) [inline sources/start/gitlab/index.md:255]: I'd cut this sentence into two shorter ones. (Just a suggestion.)
```suggestion
Congratulations, you now integrated Reliably into a repository on GitLab. Reliably can start surfacing Reliability Suggestions as you evolve your code.
```

**Review by mperrien — CHANGES_REQUESTED** [review]
I suggested a few changes, Grant.

File: sources/start/gitlab/index.md (hunk 2 above)
Comment (gtfisher) [reply sources/start/gitlab/index.md:229]: This is what I have got now, still not ideal, but a bit better:
If you select the Expand button in the Code Quality section, you will see the additional Advice from Reliably for the merge request:

**Pull request merged**